modules/hitlnext/src/migrations/v12_16_0-1608057053-add_tags_to_handoffs.ts

```diff
@@ -0,0 +1,25 @@
+import * as sdk from 'botpress/sdk'
+
+const migration: sdk.ModuleMigration = {
+  info: {
+    description: '',
+    target: 'core',
+    type: 'database'
+  },
+  up: async ({ bp, metadata }: sdk.ModuleMigrationOpts): Promise<sdk.MigrationResult> => {
+    const tableName = 'handoffs'
+    const column = 'tags'
+    const exists = await bp.database.schema.hasColumn(tableName, column)
+
+    if (!exists) {
+      await bp.database.schema.alterTable(tableName, table => table.json(column))
+    }
+
+    return {
+      success: true,
+      message: exists ? 'tags column already exists, skipping...' : 'tags column created.'
```

Comment:
Being picky, but please capitalize messages displayed to the user

Reply:
+1